app/views/schedule_rules/_schedule_rule_fields.html.haml

```diff
@@ -27,7 +27,10 @@
 
 - unless @non_highlighted_price_group_discounts&.empty?
   .well
-    %h4= t("views.schedule_rules.non_highlighted_heading")
+    - if !@highlighted_price_group_discounts.present? || @highlighted_price_group_discounts&.empty?
```

Comment:
@jossim would this work instead?
`- if @highlighted_price_group_discounts&.blank?`

Reply:
@giladshanan ahh yeah. I added this because of a spec failure and didn't clean it up